### PR TITLE
Update Nginx configuration for multishop

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Update configuration and show how to use multishop.
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| How to test?  | Configure Nginx :)

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/12082)

### CODE REVIEW

File: docs/server_config/nginx.conf.dist [hunk 1]
```diff
@@ -81,61 +81,65 @@ server {
...
+    rewrite ^/api/?(.*)$ /webservice/dispatcher.php?url=$1 last;
+
+    # Installation sandbox
+    rewrite ^(/install(?:-dev)?/sandbox)/(.*) /$1/test.php last;
+
+    # [REQUIRED EDIT] Change this block to your admin folder
+    location /admin-dev/ {
         if (!-e $request_filename) {
-            rewrite ^/.*$ /admin/index.php last; #Change this to your admin folder
+            rewrite ^/.*$ /admin-dev/index.php last;
```
Comment (Quetzacoalt91) [inline docs/server_config/nginx.conf.dist]:
```suggestion
            rewrite ^/.*$ /admin-dev/index.php last; #Change this to your admin folder
```

File: docs/server_config/nginx.conf.dist [hunk 2]
```diff
@@ -81,61 +81,65 @@ server {
...
+
     location = /robots.txt {
         auth_basic off;
         allow all;
         log_not_found off;
         access_log off;
     }
 
-    location /admin/ { #Change this to your admin folder
+    # [REQUIRED EDIT] If you are running a multishop instance
```
Comment (Quetzacoalt91) [inline docs/server_config/nginx.conf.dist]: Regarding the 2 next lines, did you mean multilang ?

Comment (PierreRambaud) [reply docs/server_config/nginx.conf.dist]: I was in a multishop context when I set up the path, but I improve this line if you want?

Comment (Quetzacoalt91) [reply docs/server_config/nginx.conf.dist]: I'm just saying this is confusing to see this "required edit" with something which seems to be unrelated. I may be wrong, this is how I understand it.

Comment (PierreRambaud) [reply docs/server_config/nginx.conf.dist]: Should be better now?

File: docs/server_config/nginx.conf.dist [hunk 3]
```diff
@@ -156,9 +160,9 @@ server {
     }
 
     # PHP FPM part
-    location ~ [^/]\.php(/|$) {
+    location ~ \.php$ {
         # Verify that the file exists, redirect to index if not
-        try_files $fastcgi_script_name /index.php$uri&$args;
+        try_files $fastcgi_script_name /index.php$uri&$args =404;
```
Comment (PierreRambaud) [inline docs/server_config/nginx.conf.dist:165]: Prevent recursive cycle if file is not found.

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @PierreRambaud

**Pull request merged**